language-reference-manual/lrm.trlc

```diff
@@ -1001,6 +1001,7 @@ section "Checks" {
           error "CB Version must be positive", version
       }
       """
+      untraced_reason = "Example code"
```

Comment:
Illegal - CI should have complained about this:
```
untraced_reason = "Example code"
^^^^^^^^^^^^^^^ language-reference-manual/lrm.trlc:1004: error: cannot overwrite frozen component untraced_reason
```

I have created https://github.com/bmw-software-engineering/trlc/issues/37 to fix that.